docs/sql-guide/functions/function-str.md

```diff
@@ -0,0 +1,88 @@
+---
+title: STR()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# STR() function
+
+`STR()` function returns the given number formatted to a String of a certain length, with given precision.
+
+## Syntax
+
+```
+STR(num_expr, lenofstr, precision)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type | Return value |
```

Comment:
The number of columns here doesn't line up with the number of things below

Reply:
Yeah, you can leave off the "return value" column, but it'd be good to include the data type for each of the args (where necessary).
So for instance the `num_expr` could be an integer or float or something else?

Reply:
Corrected!